.github/workflows/dry-run.yml

```diff
@@ -0,0 +1,53 @@
+name: dry-run
+
+on: ['push', 'pull_request']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Add conda to system path
+      run: |
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install dependencies
+      run: |
+        conda update -n base -c defaults conda
+        conda config --set safety_checks disabled
+        conda config --set channel_priority strict
+        conda env create -f environment.yml
```

Comment:
```suggestion
    - name: Add conda to system path
      uses: conda-incubator/setup-miniconda@v2
      with:
        miniconda-version: "latest"
        auto-update-conda: true
        environment-name: analysis2
        activate-environment: analysis2
        environment-file: environment.yml
```

Reply:
this can all be accomplished using this provisioned github actions for conda. you can just commit this suggested change